refactor (#327)
KxSystems/docs: docs/wp/disaster-recovery/index.md

However, the constituent parts of the utility can be run individually
and used to construct or amend a database. The basic steps are:

> 1.  Enumerate the table to be saved against the sym file ([`.Q.en`](../../ref/dotq.md#en-enumerate-varchar-cols))

Reviewer: ```suggestion
1.  Populate the table to be saved against the sym file ([`.Q.en`](../../ref/dotq.md#en-enumerate-varchar-cols))
```

Author: 'Enumerate' is the action that it is performing ( https://en.wikipedia.org/wiki/Enumerated_type#:~:text=In%20computer%20programming%2C%20an%20enumerated,or%20enumerators%20of%20the%20type. ). Cant change to 'Populate'.

Note that as part of setting a splayed table, the column `.d` file is
automatically written by kdb+. After these steps, the database can now

> be loaded into q via a simple [`\l` command](../../basics/syscmds.md#l-load-file-or-directory).

Reviewer: ```suggestion
be loaded into q using a simple [`\l` command](../../basics/syscmds.md#l-load-file-or-directory).
```

If a real-time database does go down, then it is usual practice to
trigger a failover to the secondary RDB, such that the gateway will

> only route queries to the secondary RDB while the primary is offline

Reviewer: ```suggestion
If a real-time database does go down, a failover to the secondary RDB is generally triggered. The gateway then only routes queries to the secondary RDB while the primary is offline.
```

or recovering. Failover can be signaled via [`.z.pc`](../../ref/dotz.md#zpc-close). Usually, the
handles to the secondary RDB are opened prior to any failover
occurring, and it is simply a case of pointing the query to the

> secondary RDB handle, in place of the primary RDB handle. In larger

Reviewer: ```suggestion
handles to the secondary RDB are opened prior to any failover occurring, allowing you to point the query to the secondary RDB handle in place of the primary RDB handle. In larger
```

the secondary process. The subscriber isn’t lost and could reconnect
to the secondary TP, allowing seamless transition. Clients that have

> subscribed to the tickerplant will receive a closed connection

Reviewer: ```suggestion
the secondary process. The subscriber isn’t lost and could reconnect
to the secondary TP, allowing seamless transition. Clients that have
subscribed to the tickerplant receive a closed connection
```
